documentation/Installation.md

```diff
@@ -244,13 +244,7 @@ For example, specify `conntrack-tools` instead of `conntrack`.
 **Note**:
 
 * You can install a version other than the recommended version, but it is not supported and can cause unpredictable consequences.
-* rh-haproxy18 (build provided by RedHat) is supported only for now.
-
-**Warning**: RHEL 8 does not have Python preinstalled. For `check_iaas` to work correctly, it is required to install Python on the nodes. Execute the following step before the installation procedure.
-* Install `python 3.9` from the standard RHEL repository:
-```
-# dnf install python3.9 -y
-```
+* Note: RHEL 7/8 and CentOS 7/8 are no longer supported.
```

Comment:
As I understand, OL and Rocky 8 are also not supported. By the way, why we decided to drop support for RHEL, OL and Rocky 8?